src/server.py

```diff
@@ -433,50 +434,38 @@ def run_seta_query():
     return {'failures': failures}
 
 
-@app.route("/data/setasummary/")
-@json_response
-def run_seta_summary_query():
-    query = session.query(Seta.date).distinct().all()
-    retVal = {}
-    dates = []
-    for d in query:
-        dates.append(d[0].strftime("%Y-%m-%d"))
-
-    for d in dates:
-        count = session.query(Seta.id).filter(Seta.date == d).count()
-        retVal['%s' % d] = "%s" % int(count)
-
-    return {'dates': retVal}
-
-
 @app.route("/data/setadetails/")
 @json_response
 def run_seta_details_query():
-    startDate, date = clean_date_params(request.args)
-    active = sanitize_bool(request.args.get("active", 0))
+    inactive = sanitize_bool(request.args.get("inactive", 0))
     buildbot = sanitize_bool(request.args.get("buildbot", 0))
     branch = sanitize_string(request.args.get("branch", ''))
     taskcluster = sanitize_bool(request.args.get("taskcluster", 0))
-    priority = sanitize_string(request.args.get("priority", 0))
-    head_rev = sanitize_string(request.args.get("head_rev", ''))
-    pushlog_id = sanitize_string(request.args.get("pushlog_id", ''))
+    priority = int(sanitize_string(request.args.get("priority", '1')))
+
     jobnames = JOBSDATA.jobnames_query()
-    if date == "" or date == "latest":
-        today = datetime.now()
-        date = today.strftime("%Y-%m-%d")
-    date = "%s" % date
-    query = session.query(Seta.jobtype).filter(Seta.date == date).all()
+    date = str(datetime.now().date())
+
+    if inactive == 1:
+        priority = 5
+    else:
+        priority = 1
+
+    # TODO: we can make this a variable priority in the future based on input
```

Comment:
Right now we are querying based on priority 1 (high value jobs).  Then we either return that, or the entire list of jobs - priority 1 (low value jobs).

When we have other priorities, I imagine us querying all jobs into the scope and filtering based on the input.  Maybe where priority in [1,2,3] or something like that.